From: Tzafrir Cohen <tzafrir@debian.org>
Subject: install service and init.d files on GNU/kFreeBSD

The only effect this patch has is to install the systemd service on
kFreeBSD. Not very useful, but keeps from breaking our configuration.

And yes, the HURD is missing from here as well.

--- a/Makefile
+++ b/Makefile
@@ -790,7 +790,7 @@ install-logrotate:
 	rm -f contrib/scripts/asterisk.logrotate.tmp
 
 config:
-	@if [ "${OSARCH}" = "linux-gnu" ]; then \
+	@if [ "${OSARCH}" = "linux-gnu" -o "${OSARCH}" = "kfreebsd-gnu" ]; then \
 		$(INSTALL) -d $(DESTDIR)/lib/systemd/system; \
 		./build_tools/install_subst -d contrib/asterisk.service $(DESTDIR)/lib/systemd/system/asterisk.service; \
 		if [ -f /etc/redhat-release -o -f /etc/fedora-release ]; then \
